File: core/src/main/java/com/kin/ecosystem/core/data/internal/Configuration.java
package com.kin.ecosystem.core.data.internal;

import com.kin.ecosystem.common.KinEnvironment;

public interface Configuration {

	KinEnvironment getEnvironment();

	void setEnvironment(String name);

	interface Local {

		KinEnvironment getEnvironment();

		void setEnvironment(KinEnvironment kinEnvironment);
	}

}

File: core/src/main/java/com/kin/ecosystem/core/data/internal/Configuration.kt
package com.kin.ecosystem.core.data.internal

import com.kin.ecosystem.common.KinEnvironment
import com.kin.ecosystem.common.KinTheme

interface Configuration {

    val environment: KinEnvironment

    var kinTheme: KinTheme

    interface Local {

        var kinTheme: KinTheme
    }
}
